packages/skill-api/src/core/index.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  SkillRecordingsAction,
+  SkillRecordingsHandlerParams,
+  SkillRecordingsHeader,
+} from './types'
+import {init} from './init'
+import renderPage from './pages'
+import {sendFeedbackFromUser} from './services/send-feedback-from-user'
+
+export interface OutgoingResponse<
+  Body extends string | Record<string, any> | any[] = any,
+> {
+  status?: number
+  headers?: SkillRecordingsHeader[]
+  body?: Body
+  redirect?: string
+  cookies?: any[]
+}
```

Comment:
The `SkillRecordingsHandler` below returns an outgoing response where the body can be JSON or HTML (or other bits denoted by the headers really) as well as any cookies etc to set.